### PR TITLE
Updated colorize to the latest version 1.1

### PR DESCRIPTION
**Pull request by mmotherwell**
This passes all the tests, and brings the colorize dependency up-to-date

### CODE REVIEW

**Comment by ylecuyer** [comment]
Thanks for the contribution. Before merging I'll have to adapt the CI to remove ruby 2.5 as it isn't compatible with this version of colorize (and maybe add ruby 3.3).

If you also want to handle this part in another PR you're welcome :+1:

**Comment by mmotherwell** [comment]
I remover ruby 2.5 form the .github/workflows/test.yml - is there a way I can run it in CI?

File: puma-status.gemspec
Comment (ylecuyer) [inline puma-status.gemspec:1]: could you also change the minimal required ruby version here please

Comment (mmotherwell) [reply puma-status.gemspec:1]: Done,

**Pull request merged**